src/number-input/number.component.ts

```diff
@@ -70,6 +70,7 @@ export class NumberChange {
 						type="button"
 						aria-live="polite"
 						aria-atomic="true"
+						aria-label="increment button"
```

Comment:
Should be `increment value` I think? Also we'll want to add this to the translatable text (same for decrement)